packages/schematics/angular/migrations/update-10/remove-solution-style-tsconfig.ts

```diff
@@ -35,11 +35,16 @@ export default function (): Rule {
   return (host, context) => {
     const logger = context.logger;
 
-    const files = new JSONFile(host, 'tsconfig.json').get(['files']);
-    if (!(Array.isArray(files) && files.length === 0)) {
-      logger.info('Migration has already been executed.');
+    try {
```

Comment:
I think in this case, it would be better to check if the file exists instead of using a try/catch block.